.jshintignore

```diff
@@ -1,5 +1,7 @@
 **/vendor
-node_modules
 cms/static/js/i18n/**/*.js
 lms/static/js/i18n/**/*.js
+lms/static/lms/js/build.js
+lms/static/lms/js/spec/main.js
```

Comment:
BTW, JSHint is a more flexible variant of JSLint, so we always use the former:

http://blog.scottlogic.com/2011/03/28/jslint-vs-jshint.html

Really we should be moving over to ESLint in preparation for ES2015, but I'm not sure when we'll be able to schedule that. I'm tempted to do it for the hackathon.